src/openrct2/ride/Ride.cpp

```diff
@@ -7319,101 +7319,7 @@ bool ride_has_ratings(const Ride* ride)
 
 const char* ride_type_get_enum_name(int32_t rideType)
```

Comment:
Remove the function call with `RideTypeDescriptors[rideType].EnumName` and remove this function.